src/commands/publish.ts

```diff
@@ -552,19 +585,23 @@ export async function publishMain(argv: PublishOptions): Promise<any> {
     logger.info(' ');
   }
 
-  if (argv.rev) {
-    logger.info('Not merging any branches because revision was specified.');
+  if (argv.noMerge) {
```

Comment:
Is `noMerge` a new option here?

Reply:
Nope, it was inside `handleReleaseBranch` under the name `skipMerge`: https://github.com/getsentry/craft/pull/245/files#diff-adc8934db1a646fd72f3cdda4ede76ecfc3628ec0f8dd1f37403d0121183e94aL334